### PR TITLE
Fails if no timestamps defined

### PR DESCRIPTION
**Pull request by javierg**

### CODE REVIEW

**Comment by javierg** [comment]
:shipit:

File: test/document_test.rb [hunk 1]
```diff
@@ -348,6 +364,15 @@ def setup
     assert_equal 'bar', test_foo.doc['default_test_property']
   end
 
+  test 'created at is current time' do
+    resp = {ok: true, id: "with_time/timed", rev: "FF0000"}
+    FakeWeb.register_uri :put, /http:\/\/localhost:5984\/test\/with_time%2F.+/, body: resp.to_json
+    test = WithTime.new id: "timed"
+    assert test.respond_to?(:created_at)
+    assert test.save
+    assert test.created_at
```
Comment (netlogics) [inline test/document_test.rb]: You might want to verify `created_at` is the current time rather than just presence.

Comment (seancookr) [inline test/document_test.rb]: @netlogics that is tested elsewhere

Comment (netlogics) [inline test/document_test.rb]: Seems misleading since the test name is `created at is current time`

File: test/document_test.rb [hunk 2]
```diff
@@ -316,6 +327,11 @@ def setup
     assert_equal foo.persisted?, true
   end
 
+  test 'document with no declared properties will fail' do
+    bar = BarFoo.new persist: "persisted", a:1, b:2, c:3, d:4, f:5, g:6
+    foo = FooBar.find "big_doc"
```
Comment (neojin) [inline test/document_test.rb]: are you missing an assertion on this test?

File: test/document_test.rb [hunk 3]
```diff
@@ -1,6 +1,12 @@
 require 'test_helper'
 
-class FooBar < Dolly::Document
+class BaseDolly < Dolly::Document; end
+
+class BarFoo < BaseDolly
+  property :a, :b, :c, :d, :e, :f, :g, :h, :i, :j, :k, :l, :m, :n, :persist
```
Comment (neojin) [inline test/document_test.rb]: I think you should finish the alphabet. :trollface:

File: dolly.gemspec [hunk 4]
```diff
@@ -30,5 +30,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "fakeweb", "~> 1.3.0"
   spec.add_development_dependency "factory_girl_rails", "~> 4.2"
   spec.add_development_dependency "mocha", "~> 0.4"
+  spec.add_development_dependency "timecop"
```
Comment (neojin) [inline dolly.gemspec]: where are you using TimeCop?

**Pull request merged**